packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -180,6 +180,8 @@ export interface AnchorNode extends UpPath<AnchorNode>, ISubscribable<AnchorEven
 	 * Allows access to data stored on the Anchor in "slots".
 	 * Use {@link anchorSlot} to create slots.
 	 */
+	// TODO: use something other than `any`
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	readonly slots: BrandedMapSubset<AnchorSlot<any>>;
```

Comment:
Why doesn't `unknown` work here? I tried it locally real quick and it didn't seem to complain.

Reply:
Good catch. This one works using `unknown`. Updated!